tools/tertiary-analysis/monocle3/.shed.yml

```diff
@@ -0,0 +1,20 @@
+categories:
+ - Transcriptomics
+ - RNA
+ - Statistics
+ - Sequence Analysis
+description: "De-composed monocle3 functionality tools, based on https://github.com/ebi-gene-expression-group/monocle-scripts and monocle3 0.1.2."
+long_description: |
+        scmap: A tool for trajectory analysis of single cell RNA-seq data
```

Comment:
Assume this is a copy paste error- fix to Monocle description?

Reply:
Thanks, well spotted.